pullrequests/PollPR.py

```diff
@@ -0,0 +1,721 @@
+"""
+# Aiming to be used with Hydra (Nixos) builder for Ghaf project https://github.com/tiiuae/ghaf
+
+# Activates Hydra (Ghaf docker based Hydra build system) to build new open Pull Requests
+#(for the main repo under observations)
+#
+# Rebuilds still open (and previously built) PR if there has been new changes for that PR
+# Uses Ghaf build tools from https://github.com/tiiuae/ci-public
+
+# Can be used in service mode (polling frequently changes) or cherrypicking some wished open PR
+# Keeps internal recoed which open PRs or changed PRs have been allready processed
+# (given Hydra build commands ok)
+#
+#
+#
+# Author mika.nokka1@gmail.com 13.6.2023
+"""
+
+
+# pylint: disable=invalid-name # code written and variables named before lint usage decided
+# pylint: disable=line-too-long # vscode with max 140 used
+# pylint: disable=superfluous-parens # using parenthesis in if clauses
+# pylint: disable=consider-using-with # intentional open file usage (no with)
```

Comment:
I would like too see a separate commit/PR at some point which fixes most of these things, so that pylint disables are not required. Not blocking this PR for this, but just noting.

Reply:
When using vccode, pylint and pep8 it does ok job, sees "real" issues. However if you use pylint commandline with standard settings, its traps problematic cases (like several if clauses and some of then dares to use break...). Hence if we decide to add pylint to autocheckings, we need to tune somehow the rules settings file